configure should detect version of python to use and allow override via
PYTHON; build process should change first line of script at install time to
match.

XXX file upstream bug; patch not appropriate for upstream in current form.

--- pulseaudio-6.0/src/utils/qpaeq	2015-02-12 06:10:35.000000000 -0800
+++ pulseaudio-6.0/src/utils/qpaeq	2015-06-08 12:22:20.978639398 -0700
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 #    qpaeq is a equalizer interface for pulseaudio's equalizer sinks
 #    Copyright (C) 2009  Jason Newton <nevion@gmail.com
 #
